### DISCUSSION TITLE
The glob() loader cannot be used for files in src/content ?

### DISCUSSION THREAD

**Discussion by JesseZhuang** [satnaing/astro-paper#413] [Q&A]
See https://github.com/withastro/astro/blob/main/packages/astro/src/content/loaders/glob.ts#L232C18-L232C63

Saw this warning during build. Does Astro have some design concern to not recommend using glob for `src/content`? AstroPaper may need to switch to a different api if that is the case.

**Comment by rachelslurs** [comment]
I have seen the same, but I have yet to see any problem as a result.

**Reply by JesseZhuang** [reply]
It's warning (not error), not sure what astro's future plan is.

**Reply by man0war64** [reply]
Which version of Astro are you running?
I am running 4.16.16, and the result of that warning seems to be to skip the .md files:

11:42:46 [WARN] [glob-loader] The glob() loader cannot be used for files in src/content.
11:42:46 [WARN] [glob-loader] Skipped 19 files that matched **/*.md.

**Reply by JesseZhuang** [reply]
> Which version of Astro are you running? I am running 4.16.16, and the result of that warning seems to be to skip the .md files:
> 
> 11:42:46 [WARN] [glob-loader] The glob() loader cannot be used for files in src/content. 11:42:46 [WARN] [glob-loader] Skipped 19 files that matched **/*.md.

the astropaper theme is using `glob` to load the markdown files

**Comment by obedmacallums** [comment]
I have the same warning, what does that mean?
